### ISSUE TITLE
Plugin does not work in multi-root workspaces

### ISSUE THREAD

**Issue by danielrentz** [dlevs/vscode-pattern-links#2]
Hi,

I am working with a multi-root workspace containing 10+ projects, all with their own GIT repos and `.vscode` configuration settings.

Please adapt this helpful plugin for multi-root workspaces.

https://code.visualstudio.com/docs/editor/multi-root-workspaces#_extensions

**Comment by dlevs** [comment]
Hi, what changes are needed? Does this plugin not work at the moment in your multi-root workspace? What issues are you experiencing?

**Comment by danielrentz** [comment]
Yes, it does not work at all. Please see the docs linked above. I have no idea at all in the area of VSCode plugin development :/
